Neos.Neos/Classes/Controller/Backend/MenuHelper.php

```diff
@@ -58,6 +60,12 @@ class MenuHelper
      */
     protected $iconMapper;
 
+    /**
+     * @Flow\Inject
+     * @var \Neos\Neos\Domain\Service\ContentContextFactory
```

Comment:
nitpick: This namespace could be imported for better readability

Reply:
@bwaidelich All namespaces imported for better readability